Sources/Sentry/SentryClient.m

```diff
@@ -168,11 +169,17 @@ - (SentryId *)captureError:(NSError *)error
 - (SentryEvent *)buildErrorEvent:(NSError *)error
 {
     SentryEvent *event = [[SentryEvent alloc] initWithLevel:kSentryLevelError];
-    NSString *errorCodeAsString = [NSString stringWithFormat:@"%ld", (long)error.code];
-    SentryException *sentryException = [[SentryException alloc] initWithValue:errorCodeAsString
+    NSString *exceptionValue = [NSString stringWithFormat:@"Code: %ld LocalizedDescription: %@",
```

Comment:
localizedDescription will vary by locale, is this what we want to do?

Reply:
@mitsuhiko advocated for this as we also do this on other SDKs. I changed the logic a bit so we only add it if it exists in the userInfo.